packages/oauth/tsconfig.json

```diff
@@ -30,6 +30,7 @@
   ],
   "exclude": [
     "src/**/*.spec.js",
+    "src/**/*.spec.ts",
```

Comment:
Added a few TS tests. They should be excluded in the package file. I've verified if this works by running `npm pack`.